src/components/ButtonComponents/NumberButtons/NumberButton.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 
-const NumberButton = () => {
+const NumberButton = (props) => {
   return (
-    <>
-      {/* Display a button element rendering the data being passed down from the parent container on props */}
-    </>
+    <button className ="numberbutton all">
+      {props.number/* Display a button element rendering the data being passed down from the parent container on props */}
+    </button>
```

Comment:
Again, spot on. Components should have one purpose. We have a button, it displays a number.